howso/react_aggregate.amlg

```diff
@@ -2,6 +2,7 @@
 (null
     ;Computes, caches, and returns specified details and feature prediction statistics such as Mean Decrease in Accuracy (MDA), residuals (accuracy, Mean Absolute Error),
     ; precision, recall, etc. Returns details and feature prediction stats for all features in the format of feature -> assoc stat -> value
+	;{long_running (true)}
```

Comment:
probably idempotent is false since i think you'll get diff results everytime, esp if asking for robust residuals and such?